tests/Doctrine/Tests/ORM/Functional/QueryTest.php

```diff
@@ -155,29 +156,31 @@ public function testInvalidInputParameterThrowsException()
     {
         $this->expectException(QueryException::class);
 
-        $q = $this->_em->createQuery('SELECT u FROM Doctrine\Tests\Models\CMS\CmsUser u WHERE u.name = ?');
+        $q = $this->_em->createQuery('SELECT u FROM ' . CmsUser::class . ' u WHERE u.name = ?');
         $q->setParameter(1, 'jwage');
         $user = $q->getSingleResult();
     }
 
     public function testSetParameters()
     {
-        $q = $this->_em->createQuery('SELECT u FROM Doctrine\Tests\Models\CMS\CmsUser u WHERE u.name = ?1 AND u.status = ?2');
-
         $parameters = new ArrayCollection();
         $parameters->add(new Parameter(1, 'jwage'));
         $parameters->add(new Parameter(2, 'active'));
 
-        $q->setParameters($parameters);
-        $users = $q->getResult();
+        $this->_em->createQuery('SELECT u FROM ' . CmsUser::class . ' u WHERE u.name = ?1 AND u.status = ?2')
+                  ->setParameters($parameters)
+                  ->getResult();
+
+        $this->addToAssertionCount(1);
```

Comment:
This test should probably assert on the executed query by asserting on the parameters stored in the logger